internal/checker/cloudformation/rules/logging/verify_s3_bucket_restrict_write_access.rego

```diff
@@ -0,0 +1,48 @@
+# METADATA
+# title: "Verify S3 bucket restrict write access"
+# description: "This policy ensures an S3 bucket restricts WRITE access, enhancing security through controlled public access configurations."
+# scope: package
+# related_resources:
+# - https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-s3-bucket.html
+# custom:
+#   id: CB_CFT_44
+#   severity: High
+package lib.cloudformation.CB_CFT_44
+import future.keywords.in
+
+resource := "AWS::S3::Bucket"
+
+is_valid {
+	some resources in input.Resources
+	resources.Type == resource
+}
+
```

Comment:
update logic to

pass[resources] {
	some resources in input.Resources
	is_valid
	count(fail) == 0
}

fail[block] {
	is_valid
	some resources in input.Resources
	resources.Type == resource
	block := resources.Properties
	block.AccessControl == "PublicReadWrite"
}